src/js/player.js

```diff
@@ -242,6 +249,126 @@ class Player extends Component {
     return el;
   }
 
+  width(value) {
+    return this.dimension('width', value);
+  }
+
+  height(value) {
+    return this.dimension('height', value);
+  }
+
+  dimension(dimension, value) {
+    let property = dimension + '_';
+
+    if (value === undefined) {
+      return this[property] || 0;
+    }
+
+    if (value === '') {
+      // If an empty string is given, reset the dimension to be automatic
+      this[property] = undefined;
+    } else {
+      this[property] = parseFloat(value);
```

Comment:
We shouldn't set property to `NaN` (which would happen if the `isNaN` check is true below).